cabin/identity.go

```diff
@@ -135,7 +137,7 @@ func (id *Identity) MaintainAnnouncement(selfcheck bool) (changed bool, err erro
 }
 
 // MaintainStatus maintains the Hub's Status and returns whether there was a change that should be communicated to other Hubs.
-func (id *Identity) MaintainStatus(lanes []*hub.Lane, selfcheck bool) (changed bool, err error) {
+func (id *Identity) MaintainStatus(lanes []*hub.Lane, load int, selfcheck bool) (changed bool, err error) {
 	id.Lock()
 	defer id.Unlock()
```

Comment:
I'd rather use a pointer here to signal that this might be optional instead of testing for `>= 0` and passing -1. For the "quickly pass 0" part, you can use it like `MaintainStatus(lanes, new(int), true)`